ocw_data_parser/ocw_data_parser.py

```diff
@@ -315,6 +317,16 @@ def compose_course_features(jsons, course_pages):
                         course_feature["ocw_feature_url"] = (
                             "./resolveuid/" + page["uid"]
                         )
+                        course_feature["course_feature_tag"] = next(
+                            (
+                                x["course_feature_tag"]
+                                for x in course_feature_tags
+                                if x["ocw_feature"] == course_feature["ocw_feature"]
+                                and x["ocw_subfeature"]
+                                == course_feature["ocw_subfeature"]
```

Comment:
if the suggestion to use constants is implemented:
```python
                        course_feature[ATTR_TAG] = next(
                            (
                                x[ATTR_TAG]
                                for x in course_feature_tags
                                if x[ATTR_FEATURE] == course_feature[ATTR_FEATURE]
                                and x[ATTR_SUBFEATURE]
                                == course_feature[ATTR_SUBFEATURE]

Reply:
PS If you decide to stick with a json file, I think the `setup.py` file would need to be modified to make sure it's included in the package.

Reply:
These are all good points, I think we should lay it out as you've described